pages/_app.js

```diff
@@ -153,9 +154,31 @@ if (process.env.NEXT_PUBLIC_SENTRY_DSN) {
 }
 
 export default function YieldScanApp({ Component, pageProps, err }) {
+	const router = useRouter();
+
 	useEffect(() => {
 		tawkTo(process.env.NEXT_PUBLIC_TAWK_PROP_ID);
 	}, []);
+
+  useEffect(() => {
+    Fathom.load(process.env.NEXT_PUBLIC_FATHOM_ID, {
+      includedDomains: ["yieldscan.app"],
```

Comment:
@Lord-of-Codes  why not include the `dev.yieldscan.app` here?

Reply:
> @Lord-of-Codes why not include the `dev.yieldscan.app` here?
Yes, it should be.